.github/workflows/python_poetry_main.yml

```diff
@@ -49,6 +49,8 @@ jobs:
         run: poetry run black --check ./
       - name: Check types
         run: poetry run mypy --strict .
+      - name: Check imports
+        run: poetry run isort **/*.py -c -v
```

Comment:
I was wrong about ability of isort to check unused imports. pyflakes and autoflake is able to do that.

https://github.com/PyCQA/pyflakes
https://github.com/myint/autoflake

however, some people say that is not safe, but imo any setup that is not clearly visible is not a good practice. It is always possible to add # ignore: something is it is necessary.

psf/black#86
https://stackoverflow.com/questions/57275978/example-of-remove-unused-imports-can-have-bad-side-effects-in-python

Reply:
will add it in separate mr